core/changestream/subscription.go

```diff
@@ -43,6 +47,9 @@ func (o SubscriptionOption) Filter() func(ChangeEvent) bool {
 // Namespace returns a SubscriptionOption that will subscribe to the given
 // namespace.
 func Namespace(namespace string, changeMask ChangeType) SubscriptionOption {
+	if changeMask == 0 {
+		panic(fmt.Errorf("unexpected changeMask value: 0"))
```

Comment:
noooooooo - never panic in production code, and especially not for parameter validation errors

Reply:
The alternative is to subscribe to all. I'm not a fan of returning error from the namespace.